lib/config.js

```diff
@@ -25,3 +25,8 @@ cfg.routes = {
   registration: '/authn/token/client/registration',
   recovery: '/authn/token/recovery'
 };
+
+cfg.delay = {
+  jitter: 0, // randomized delay value in ms
+  hmacKey: 'a test key'
```

Comment:
```suggestion
  hmacKey: 'exampleKey'
```

We may also want to add a comment here explaining what this does and how it should be used in dev vs in production.